Ontology, visual model, and vocabularies for modeling theater showtimes metadata (DRK UC3) (#17)
Fraunhofer-FIT-DSAI/drk-information-model: vocabs/README.md

# Controlled Vocabularies for Representing Enumerations

- For modeling Enumerations (ENUMs) in semantic metadata schemas (and later using them while instantiating these

>   metadata models in the DRK Knowledge Graph), it would be preferable use URIs from existing controlled vocabularies (

Reviewer: ```suggestion
  metadata models in the DRK Knowledge Graph), it is preferable use URIs from existing controlled vocabularies (
```
